Formula/z/zigmod.rb

```diff
@@ -43,7 +43,7 @@ def install
   end
 
   test do
-    (testpath/"zig.mod").write <<~EOS
+    (testpath/"zigmod.yml").write <<~YAML
```

Comment:
why this change, this should be `zig.mod`?

Reply:
Latest versions use `zigmod.yml` - https://github.com/nektro/zigmod/commit/2431edfc4f3987c4c31909713ccf643d2253b02d

Reply:
I see, thanks!